private/buf/bufformat/testdata/proto2/option/v1/option_message_field.golden.proto

```diff
@@ -20,7 +20,7 @@ message Test {
   repeated Simple r = 4;
   map<string, int32> m = 5;
 
-  optional bytes b = 6 [default = "\x00\x01\x02\x03\x04\x05\x06\afubar!"];
+  optional bytes b = 6 [default = "\0\1\2\3\4\5\6\7fubar!"];
```

Comment:
This actually uncovered a small edge case from before - the original input was unnecessarily transformed, but now it's preserved.